src/firebase/user/user.cr

```diff
@@ -18,25 +21,63 @@ module Google
       @[JSON::Field(key: "timeZone")]
       getter time_zone : String?
 
+      # fields left out
+      # dateOfBirth
+      # passwordHash
+      # salt
+      # version
```

Comment:
If you have a link to the documentation that defines this model, it would be worth putting that in the doc comment above the object. Then you can also include these fields as a markdown list and it will show up when `crystal tool docs` is run.